common/country_types/hfe_country_types_pirate.txt

```diff
@@ -21,14 +23,20 @@ hfe_pirate = {
 	enforces_borders = no
 	share_communications = no
 	attack_unarmed_orbital_stations = yes
-	can_have_their_comms_shared = no
+	can_have_their_comms_shared = yes
```

Comment:
I see no reason why pirate should share communications

Reply:
It's meant so that when a nation trades communications, they also add "oh, and there are some pirates we're dealing with in this sector, you should watch out".

Reply:
Pirate contact function from event.
No need in this option.

Reply:
Not always. I have seen uncontacted pirates in the game before. IIRC, they are not affixed to the Day Zero settings.

Reply:
Pirates do not have Day Zero settings. It is important that they be met over a period of time.

Reply:
Day zero means a block of default establish communication project.
Pirates getting communications vie event.
Also, i have no idea with vanilla communications sharing work with non default country types, probably not.

Reply:
It does. You can spread contact with the Ju Ahets and the Kad Republic. If you have met either one and then share communications with someone else, they get to meet the Ju Ahets and friends as well.